src/cli.rs

```diff
@@ -776,6 +776,11 @@ impl clap::FromArgMatches for Exec {
                     .get_occurrences::<String>("exec_batch")
                     .map(CommandSet::new_batch)
             })
+            .or_else(|| {
```

Comment:
Could this be implemented in a way that would allow using it along with exec or exec-batch?